styles/components/body-content.scss

```diff
@@ -0,0 +1,57 @@
+.body-content {
+  padding: $gutter;
+  text-align: left;
+}
+
+.grid-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container {
```

Comment:
This container is doing WAY too much. Refer back to the rock solid lesson notes in TIYO.